resources/HtmlTestPage.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test homework</title>
+</head>
+
+
+
+<body>
+
+
+<h1>Orders for delivery</h1>
+</body>
+
+<style>
+<head>
+<body>
+table {
+  font-family: arial, sans-serif;
+  width: 200%;
+}
+
+td, th {
+  border: 0.01px solid #dddddd;
+  text-align: left;
+  padding: 20px;
+}
+
+td  {
+  background-color: green;
+}
+
+</style>
+<body>
+
+
+<h2>table</h2>
+
+<table >
+    <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Address</th>
+        <th>Contact</th>
+    </tr>
+    <tr>
+        <td>001</td>
+        <td>Ivan I.V</td>
+        <td>Microraion Zelenyi d. 15 kv. 97</td>
+        <td>+375293654512</td>
+    </tr>
+    <tr>
+        <td>002</td>
+        <td>Dima D.D</td>
+        <td>Microraion Zelenyi d. 55 kv. 89</td>
+        <td>+37529894562136</td>
+    </tr>
+    <tr>
+        <td>003</td>
+        <td>Sasha S.S</td>
+        <td>Microraion Zelenyi d. 55 kv. 756</td>
+        <td>+375442589632</td>
+    </tr>
+    <tr>
+        <td>004</td>
+        <td>Nik F.F</td>
+        <td>Microraion Zelenyi d. 15 kv. 97</td>
+        <td>+375335648798</td>
+    </tr>
+</table>
+
+</body>
+</html>
+
+
+<h2>Delivery type</h2>
+
+
+<form action="/action_page.php">
+    <input type="radio" id="Bike" name="fav_language" value="Bike">
+    <label for="Bike">Bike</label><br>
+    <input type="radio" id="Car" name="fav_language" value="Car">
+    <label for="Car">Car</label><br>
+    <input type="radio" id="Foot" name="fav_language" value="Foot">
+    <label for="Foot">Foot</label>
+
+<h1>Addition to the order</h1>
+
+<form action="/action_page.html">
+    <input id="extra1" name="extra1" type="checkbox" value="sauce">
+    <label for="extra1">Drink</label><br>
+    <input id="extra2" name="extra2" type="checkbox" value="appliances">
+    <label for="extra1">Extra ketchup</label><br>
+    <button type="button">Send</button>
+</form>
+
+
+
+<p>Time of delivery:</p>
+
+
+
+<label for="Time">Order waiting time</label>
+
+<select id="Time">
+    <option value="12">12:00 - 12:30</option>
+    <option value="13">13:00 - 13:30</option>
+    <option value="18">18:00 - 18:10</option>
+    <option value="19">19:00 - 19:30</option>
+</select>
+
+
+    <p>Click to confirm your order has been shipped</p>
+
+    <button type="button">Send an order</button>
+
+<h2>Burger</h2>
+<img src="img.jpg">
+
+
+<p><a href="https://burger-king.by/">Burger-king</a></p>
```

Comment:
There's an error in HTML code, please fix it
<img width="1652" alt="image" src="https://user-images.githubusercontent.com/14195681/177398900-f609ae2c-c654-4895-af63-fb2b0eb92c38.png">